configs/11.0/packages/tcmalloc/sources

```diff
@@ -41,3 +41,20 @@ ATSRC_PACKAGE_TARS=""
 ATSRC_PACKAGE_MAKE_CHECK=""
 ATSRC_PACKAGE_DISTRIB=no
 ATSRC_PACKAGE_BUNDLE=toolchain_extra
+
+
+atsrc_get_patches ()
+{
+	# Fix ucontext issue.
+	# Remove this when gperftools PR #908 is merged.
+	at_get_patch \
+		https://github.com/tuliom/gperftools/commit/99089d65d11f.patch \
+		67e4d8d1c2876c25f94ffb257392325b || return ${?}
```

Comment:
@rzinsly  yes. I mistake with patchwork. I didn't see that PR was not merged yet so I guess there's no problem using your repo @tuliom. If the PR was already merged I will recommend to generate the patch from the gperftools repo but that's not the case.

https://github.com/gperftools/gperftools/pull/908